src/core/test/mock/pile.sol

```diff
@@ -34,6 +35,10 @@ contract PileMock {
         return totalBalanceReturn;
     }
 
+    function debtOf(uint loan) public returns (uint) {
+        return debtOfReturn;
```

Comment:
why not just return loan.debt?

Reply:
because loan.debt is outdated and not existing anymore.

We only store pie.
I think somebody already adjusted the method in an unmerged PR

Reply:
I think we should update this mock `Pile` contract to reflect the new `Loan` struct